Task_1/Task_1/Class1.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public struct Range
+    {
+        private readonly int _a; // Левая граница диапазона
+        private readonly int _b; // Правая граница диапазона
+
+        public int A => _a;
+        public int B => _b;
+        public int Count => B - A; // Количество чисел в диапазоне
+
+        public Range(int a, int b)
+        {
+            if (b <= a)
+                throw new ArgumentException("Значение B должно быть больше A.");
+            if (a >= b)
+                throw new ArgumentException("Значение A должно быть меньше значения B.");
+
+            _a = a;
+            _b = b;
+        }
```

Comment:
Конструктор написан неверно. Невозможно создать пустой диапазон [0; 0).
Из-за этого не проходит тест `OperatorAndDisjointRangesEmptyRange()`
```suggestion
        public Range(int a, int b)
        {
           if(a < b || a == 0 && b == 0)
            {
                _a = a;
                _b = b;
                return;
            }

            throw new ArgumentException("Должно быть A < B или A = B = 0");
        }
```